modules/setting/setting.go

```diff
@@ -907,6 +910,14 @@ func NewContext() {
 		}
 	}
 
+	if RunUser == "root" {
```

Comment:
We can't entirely (Windows is the case where it is certain to not be "root"), however windows is out of scope of this PR. This purpose of this PR wasn't to be exhaustive of all the possibilities, just to prevent me from being lazy and running things as root where I could mess up file permissions 😆

Reply:
I'd stick a GOOS check here and exclude dozers from this check.
Then just use os.GetUID() == 0 instead of checking username.

Reply:
we dont need to check goos on windows we will get a -1